skimage/io/tests/test_pil.py

```diff
@@ -67,7 +67,7 @@ def test_imread_separate_channels():
     with NamedTemporaryFile(suffix='.tif') as f:
         fname = f.name
 
-    imsave(fname, x)
+    imsave(fname, x, photometric='rgb', planarconfig='separate')
```

Comment:
Thanks for the fix! See https://github.com/scikit-image/scikit-image/issues/6558#issuecomment-1273074306. What do you think @kushal-gopal?

Reply:
Thanks @lagru. Yes it makes sense to keep the default arguements.
PS - Wanted to ask what do you mean by round tripping here, didn't understand that in this context.

Reply:
With "roundtripping" I meant that the image is saved in this line and read in the very next line. The test checks the properties (in this case the axis order) of the object after it has been saved and loaded again.